packages/sdk/snet/sdk/__init__.py

```diff
@@ -1,3 +1,13 @@
+import google.protobuf.internal.api_implementation
+
+google.protobuf.internal.api_implementation.Type = lambda: 'python'
```

Comment:
Last year I've faced this annoying issue...
I remember that setting:
`os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "python"` did the trick ([Reference](https://github.com/protocolbuffers/protobuf/issues/3002)).
But I don't know how this is being handle now.

Reply:
This line essentially does the same thing.

But at the same time, it doesn't completely solve the problem. Now the symbols database is used both in the cpp implementation, and in the pure python implementation.

If you look at the compiled proto files you'll find a bunch of `_sym_db.RegisterMessage(...)`, which is what I'm talking about. That's why I'm essentially killing this function.

Going forward I plan to use reflection to override the function to simply use another name for the key in the database instead of the service's descriptor.

Reply:
Got it, thanks!